slovak/marsmelaci.md

```diff
@@ -1,2 +1,6 @@
-Jedno nevšedné ráno sa zobudíš v divnej miestnosti, ktorá má steny z marshmallow na to, že máš v nose kyslú rybičku.
```

Comment:
Sentence structer rebuilt - raw translation ".... which has a marshmallow walls for, you have a sour fish in nose."